### PR TITLE
GLP-602 GLP-601 add document type: ApBundleAdditional

### PR DESCRIPTION
**Pull request by JDAMOY**
add document type: ApBundleAdditional @yhl10000 @rcqing

### CODE REVIEW

File: src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd [hunk 1]
```diff
@@ -2151,6 +2152,123 @@
...
+              <ccts:Definition>An association to Proforma Reference.</ccts:Definition>
+              <ccts:Cardinality>1..1</ccts:Cardinality>
+              <ccts:ObjectClass>Fapiao Info Message</ccts:ObjectClass>
+              <ccts:PropertyTerm>Proforma Reference</ccts:PropertyTerm>
+              <ccts:AssociatedObjectClass>Proforma Reference</ccts:AssociatedObjectClass>
+            </ccts:Component>
+          </xsd:documentation>
+        </xsd:annotation>
+      </xsd:element>
+      <xsd:element ref="cbc:FapiaoCodeValue" minOccurs="0" maxOccurs="1">
```
Comment (yhl10000) [inline src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd]: minOccurs="1"

File: src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd [hunk 2]
```diff
@@ -2151,6 +2152,123 @@
...
+              <ccts:PropertyTerm>Fapiao Code Value</ccts:PropertyTerm>
+              <ccts:RepresentationTerm>Text</ccts:RepresentationTerm>
+              <ccts:DataType>Text. Type</ccts:DataType>
+              <ccts:AlternativeBusinessTerms>Fapiao Code Value</ccts:AlternativeBusinessTerms>
+              <ccts:Examples>3420001, 3273666, HSJJD-213</ccts:Examples>
+            </ccts:Component>
+          </xsd:documentation>
+        </xsd:annotation>
+      </xsd:element>
+      <xsd:element ref="cbc:FapiaoNumValue" minOccurs="0" maxOccurs="1">
```
Comment (yhl10000) [inline src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd]: minOccurs="1"

File: src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd [hunk 3]
```diff
@@ -2151,6 +2152,123 @@
...
+              <ccts:ComponentType>BBIE</ccts:ComponentType>
+              <ccts:DictionaryEntryName>Classification Category. Code Value. Text</ccts:DictionaryEntryName>
+              <ccts:Definition>The Classification Category value.</ccts:Definition>
+              <ccts:Cardinality>0..1</ccts:Cardinality>
+              <ccts:ObjectClass>Fapiao Info Message</ccts:ObjectClass>
+              <ccts:PropertyTerm>Fapiao Code Value</ccts:PropertyTerm>
+              <ccts:RepresentationTerm>Text</ccts:RepresentationTerm>
+              <ccts:DataType>Text. Type</ccts:DataType>
+              <ccts:AlternativeBusinessTerms>Fapiao Code Value</ccts:AlternativeBusinessTerms>
+              <ccts:Examples>3420001, 3273666, HSJJD-213</ccts:Examples>
```
Comment (yhl10000) [inline src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd]: HSJJD-213 ？

File: src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd [hunk 4]
```diff
@@ -2151,6 +2152,123 @@
...
+              <ccts:ComponentType>BBIE</ccts:ComponentType>
+              <ccts:DictionaryEntryName>Classification Category. Code Value. Text</ccts:DictionaryEntryName>
+              <ccts:Definition>The Classification Category value.</ccts:Definition>
+              <ccts:Cardinality>0..1</ccts:Cardinality>
+              <ccts:ObjectClass>Fapiao Info Message</ccts:ObjectClass>
+              <ccts:PropertyTerm>Fapiao Num Value</ccts:PropertyTerm>
+              <ccts:RepresentationTerm>Text</ccts:RepresentationTerm>
+              <ccts:DataType>Text. Type</ccts:DataType>
+              <ccts:AlternativeBusinessTerms>Fapiao Num Value</ccts:AlternativeBusinessTerms>
+              <ccts:Examples>3420001, 3273666, HSJJD-213</ccts:Examples>
```
Comment (yhl10000) [inline src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd]: HSJJD-213 ？

File: src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd [hunk 5]
```diff
@@ -2151,6 +2152,123 @@
...
+              <ccts:ComponentType>BBIE</ccts:ComponentType>
+              <ccts:DictionaryEntryName>Classification Category. Code Value. Text</ccts:DictionaryEntryName>
+              <ccts:Definition>The Classification Category value.</ccts:Definition>
+              <ccts:Cardinality>0..1</ccts:Cardinality>
+              <ccts:ObjectClass>Fapiao Info Message</ccts:ObjectClass>
+              <ccts:PropertyTerm>Fapiao Verification Code Value</ccts:PropertyTerm>
+              <ccts:RepresentationTerm>Text</ccts:RepresentationTerm>
+              <ccts:DataType>Text. Type</ccts:DataType>
+              <ccts:AlternativeBusinessTerms>Fapiao Verification Code Value</ccts:AlternativeBusinessTerms>
+              <ccts:Examples>3420001, 3273666, HSJJD-213</ccts:Examples>
```
Comment (yhl10000) [inline src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd]: if we don't know, i think we can remove this line.

File: src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd [hunk 6]
```diff
@@ -2151,6 +2152,123 @@
...
+              <ccts:ComponentType>BBIE</ccts:ComponentType>
+              <ccts:DictionaryEntryName>Classification Category. Code Value. Text</ccts:DictionaryEntryName>
+              <ccts:Definition>The Classification Category value.</ccts:Definition>
+              <ccts:Cardinality>0..1</ccts:Cardinality>
+              <ccts:ObjectClass>Fapiao Info Message</ccts:ObjectClass>
+              <ccts:PropertyTerm>Fapiao Amount Value</ccts:PropertyTerm>
+              <ccts:RepresentationTerm>Text</ccts:RepresentationTerm>
+              <ccts:DataType>Text. Type</ccts:DataType>
+              <ccts:AlternativeBusinessTerms>Fapiao Amount Value</ccts:AlternativeBusinessTerms>
+              <ccts:Examples>3420001, 3273666, HSJJD-213</ccts:Examples>
```
Comment (yhl10000) [inline src/main/resources/org/oasis-open/ubl/schemas/common/UBL-CommonAggregateComponents-2.0.xsd]: HSJJD-213 ?

File: src/main/resources/org/oasis-open/ubl/schemas/maindoc/UBL-ApBundleAdditional-2.0.xsd [hunk 7]
```diff
@@ -226,68 +226,32 @@
...
                     <xsd:documentation>
                         <ccts:Component>
                             <ccts:ComponentType>BBIE</ccts:ComponentType>
                             <ccts:DictionaryEntryName>Classification Category. Code Value. Text</ccts:DictionaryEntryName>
                             <ccts:Definition>The Classification Category value.</ccts:Definition>
                             <ccts:Cardinality>0..1</ccts:Cardinality>
-                            <ccts:ObjectClass>Fapiao 4 Fields Message</ccts:ObjectClass>
-                            <ccts:PropertyTerm>Fapiao Verification Code Value</ccts:PropertyTerm>
+                            <ccts:ObjectClass>ApBundle Additional Message</ccts:ObjectClass>
+                            <ccts:PropertyTerm>Logistics Code qValue</ccts:PropertyTerm>
```
Comment (yhl10000) [inline src/main/resources/org/oasis-open/ubl/schemas/maindoc/UBL-ApBundleAdditional-2.0.xsd]: please refer https://baiwangtradeshift.atlassian.net/wiki/display/CEP/UBL+Mapping

File: src/main/resources/org/oasis-open/ubl/schemas/maindoc/UBL-ApBundleAdditional-2.0.xsd [hunk 8]
```diff
@@ -226,68 +226,32 @@
...
+                            <ccts:ObjectClass>ApBundle Additional Message</ccts:ObjectClass>
+                            <ccts:PropertyTermQualifier>FapiaoInfo</ccts:PropertyTermQualifier>
+                            <ccts:PropertyTerm>FapiaoInfo</ccts:PropertyTerm>
+                            <ccts:AssociatedObjectClass>FapiaoInfo</ccts:AssociatedObjectClass>
                         </ccts:Component>
                     </xsd:documentation>
                 </xsd:annotation>
             </xsd:element>
-            <xsd:element ref="cbc:FapiaoVerificationCodeValue" minOccurs="0" maxOccurs="1">
+            <xsd:element ref="cbc:LogisticsCodeValue" minOccurs="0" maxOccurs="1">
```
Comment (yhl10000) [inline src/main/resources/org/oasis-open/ubl/schemas/maindoc/UBL-ApBundleAdditional-2.0.xsd]: only one?

**Comment by yhl10000** [comment]
finished first round review. @JDAMOY

**Comment by JDAMOY** [comment]
pls review again @yhl10000

**Review by yhl10000 — APPROVED** [review]
🎃

**Pull request merged**